pages/login.tsx

```diff
@@ -6,12 +6,23 @@ import {
     Input,
     FormControl,
 } from "@chakra-ui/react";
+import { useState } from "react";
+import useLocalStorage from "@utils/useLocalStorage";
+import { signIn } from "next-auth/client";
 
 /**
  * This is the page that a user will use to either login or register
  * to the SDC platform
  */
 export default function Login(): JSX.Element {
+    // hook for storing the email
+    const [email, setEmail] = useState("");
+    // save the email into localstorage for email verification page
+    const [, setLocalStorageEmail] = useLocalStorage(
```

Comment:
Can you explain the use of local storrage? Is it just for state management?

Reply:
If so, for state management, we will use unstated-next. You can find documentation about it in the README